autoinstall_pop_os.sh

```diff
@@ -8,51 +8,52 @@ echo "	 / ___ / /_/ / / / / /_/ / /___/ _, _/ / / /____/ // ___ | "
 echo "	/_/  |_\____/ /_/  \____/\____/_/ |_| /_/_____/___/_/  |_| "
 echo " "                                                          
 
-
+DIR=crylia
 # dependencies for meson, ninja, rofi, awesome and all extra optional packages
-sudo apt -y install meson ninja-build cmake cmake-data pkg-config papirus-icon-theme xorg build-essential git make autoconf automake flex bison check go-md2man doxygen cppcheck ohcount pulseaudio-utils upower bluez xorg xfce4-power-manager playerctl lightdm light-locker libxcb-ewmh-dev libxcb-xfixes0-dev libev-dev libxcb-damage0-dev libxcb-sync-dev libxcb-composite0-dev libxcb-present-dev uthash-dev libconfig-dev libgl-dev alacritty bison flex check
+sudo apt -y install meson ninja-build cmake cmake-data pkg-config papirus-icon-theme xorg build-essential git make autoconf automake flex bison check go-md2man doxygen cppcheck ohcount pulseaudio-utils upower bluez xorg xfce4-power-manager playerctl lightdm light-locker libxcb-ewmh-dev libxcb-xfixes0-dev libev-dev libxcb-damage0-dev libxcb-sync-dev libxcb-composite0-dev libxcb-present-dev uthash-dev libconfig-dev libgl-dev ncurses-term bison flex check
```

Comment:
Could you please change it back to alacritty? Its the Terminal that my theme uses and where I provide a config for.

Reply:
Oh right, I have removed the terminal alacrity because when I tried installing it it says thats not there and suggested this package. And when I searched on [pkgs.org](https://pkgs.org) there wasn't a version for Ubuntu

Reply:
I just looked on the website and github page and you need to install it with cargo

Reply:
Urgh, Ubuntu...

I think the best solution is to just dump Alacritty in favor of Kitty, since I use Kitty now and I got a config for it in the dev branch already. Kitty should be available, if you want you can add the config in this PR or I'll do it later in a seperate one after the merge.

(please change it to kitty, then it can be merged :+1: )

Reply:
> Urgh, Ubuntu...
> 
> I think the best solution is to just dump Alacritty in favor of Kitty, since I use Kitty now and I got a config for it in the dev branch already. Kitty should be available, if you want you can add the config in this PR or I'll do it later in a seperate one after the merge.
> 
> (please change it to kitty, then it can be merged +1 )

welp, i could also some lines that checks if rust is installed and install alacritty with cargo

Reply:
> you want to change to both arch and pop_os installer?

yes please

Reply:
so i add to backup both kitty and alacritty or only kitty?

Reply:
ok changed it to kitty and removed alacritty, only the backup i didn't do